src/main/java/com/solvdLaba/airport/FlightState.java

```diff
@@ -0,0 +1,19 @@
+package com.solvdLaba.airport;
+
+public enum FlightState {
+
+    SCHEDULED,
+    GATE_OPEN,
+    GATE_CLOSE,
+    START_REGISTERING,
+    START_BOARDING,
+    START_GATING,
+    IN_FLIGHT,
+    ARRIVED,
+    DELAYED,
+    CANCELED;
+
+    public void setFlightState(Flight flight) {
```

Comment:
Reverse it)
In your Flight class you should have a
```
setState(FlightState state) {
  this.state = state;
}
```

Reply:
Done